### DISCUSSION TITLE
Paste in a specific folder

### DISCUSSION THREAD

**Discussion by martoiu** [DominikPieper/obsidian-ReadItLater#139] [Ideas]
I would like to request a new feature. I often paste from URLs, and it would be helpful to create a new note directly in a specific folder. For instance, having a context menu in a folder (activated by right-clicking on the folder) would be handy. Alternatively, a button, shortcut, or hotkey to paste into the actively focused folder would also be beneficial.

**Comment by mariavd** [comment]
I have a workaround for this. I add tags to the articles manually and the plugin Auto Note Mover puts the article in the correct place if the tag matches the set rules.

**Comment by St4ndardDev1anT** [comment]
+1

At the very least, it'd be super useful if we had the ability to designate different folders for each content type the plugin already supports (YouTube, Twitter, etc.)

**Reply by adamluckdev** [reply]
You can achieve this with new [directories template variables](https://github.com/DominikPieper/obsidian-ReadItLater?tab=readme-ov-file#inbox-and-assets-directories).
